saltproc/tests/saltproc_test.py

```diff
@@ -79,26 +103,45 @@ def test_write_mat_file():
                 assert line.rstrip() == solution
 
 
-def test_process_fuel():
-    saltproc.process_fuel()
-    h1 = saltproc.bu_adens_db_0[saltproc.current_step,
-                                saltproc.find_iso_indx('H1')]
-    assert saltproc.bu_adens_db_0[saltproc.current_step, 0] == pytest.approx(
-        1.8811870e-09, 1e-6)
-    h1 = saltproc.find_iso_indx('H1')
-    assert saltproc.bu_adens_db_0[saltproc.current_step, h1] == pytest.approx(
-        1.8811870e-09, 1e-6)
-    assert saltproc.bu_adens_db_0[saltproc.current_step, 1] == pytest.approx(
-        1.0529505e-10, 1e-7)
-    h2 = saltproc.find_iso_indx('H2')
-    assert saltproc.bu_adens_db_0[saltproc.current_step, h2] == pytest.approx(
-        1.0529505e-10, 1e-7)
-
-
-def test_process_th():
```

Comment:
why is this test deleted? Is it because this PR has removed this functionality? Has equivalent functionality replaced this? Will previous simulations run properly?

Tests shouldn't just be deleted because they don't fit the current mood. A codebase need to move forward, not back and forth.